### DISCUSSION TITLE
New Source: Google AdSense

### DISCUSSION THREAD

**Discussion by nwayne94** [airbytehq/airbyte#33148] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
Google AdSense --> S3 and/or Redshift

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? 
Our marketing team currently uses Google AdSense along with various other platforms. Each platform presents its data through its own method which causes our marketing team to try and hand stitch various platforms into a single meaningful picture. This integration would allow us to pull Google AdSense data into our lake, like we are doing with other sources through Airbyte, so that we have complete control over making the sources mesh together more cleanly.

* How often do you want to run syncs?
~ once every 24 hours
* If this is an API source connector, which entities/endpoints do you need supported?
https://developers.google.com/adsense/management

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
Our marketing team will continue to use Google tools to locate the relevant insights, but it will remain separated from the rest of our data lake that already houses data from various other sources through Airbyte

## Are you willing to submit a PR?
Yes, once the new year starts and we wrap up some of our current projects

**Comment by arnaudselva** [comment]
@marcosmarxm hello, any way to add the hackathon label to this one? I can work on it

[a comment by an automated account is not shown]
